src/types/driving-permit-details.ts

```diff
@@ -0,0 +1,49 @@
+import { body, ValidationChain } from "express-validator";
+import { nameof } from "./util/nameof";
+
+export default interface DrivingPermitDetails {
+  personalNumber?: string;
+  fullAddress?: string;
+  issueBy?: string;
+  issueNumber?: string;
+  issuingCountry?: string;
+  expiryDate: string;
+  issueDate?: string;
+  validFrom?: string;
+  validUntil?: string;
+}
+
+export const generateDrivingPermitPropertyValidators = (
+  parent?: string
+): ValidationChain[] => {
+  const prefix = parent ? `${parent}.` : "";
```

Comment:
I don't know if this is a good pattern, but it was the best way I could think of. Most of the time it is set, because you have to describe the full chain of properties from the root object. The only case it's unset is for the root object (so ConfigRequest). ConfigRequest could simply not take the optional parameter and all the other types could require it. The reason for duplicating the prefix logic for all the types is just for a consistent pattern. Potentially useful If you had a type that could appear in different locations in a request body.

Reply:
Cool looks like this has been removed